extra-electronics/apicula/autobuild/prepare

```diff
@@ -0,0 +1,7 @@
+abinfo "Extracting prebuilt chip databases ..."
+for i in apycula/GW1N-1.pickle apycula/GW1N-4.pickle apycula/GW1N-9.pickle apycula/GW1NS-2.pickle
```

Comment:
Absolute path here.

Reply:
I don't think absolute path applies here, because this is filenames under zip file.